Tests/Transformer/ModelToElasticaAutoTransformerTest.php

```diff
@@ -473,6 +483,54 @@ public function testThatNestedObjectsDontNeedAnIdentifierField()
         ), $data['subWithoutIdentifier']);
     }
 
+    public function testNestedTransformHandlesSingleObjects()
+    {
+        $transformer = $this->getTransformer();
+        $document    = $transformer->transform(new POPO(), array(
+            'upper' => array(
+                'type' => 'nested',
+                'properties' => array('name' => null)
```

Comment:
For some reason the original code `'properties' => array('name' => '~')` doesn't work on PHP 5.3, perhaps @stof has any idea?